tst/schedulers/test_botorch.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+from examples.training_scripts.height_example.train_height import (
+    height_config_space,
+    METRIC_ATTR,
+)
+
+from syne_tune import Tuner, StoppingCriterion
+from syne_tune.backend import LocalBackend
+from syne_tune.optimizer.baselines import BoTorch
+import pytest
+
+
+@pytest.mark.timeout(30)
+def test_botorch_reproducible():
```

Comment:
I'd prefer not to have this test, unless BoTorch explicitly guarantees reproducible outputs. For our built-in
BO schedulers, we do not, and I am almost sure they do not either.

I'd like to avoid having tests that fail later for random reasons.

Reply:
https://github.com/pytorch/botorch/issues/1190 They expect the results to be reproducible.

Reply:
I understand the hesitation, though we seem to be able to make it reproducible now, so at least then this test would let us know when that changes, and if it's not on syne-tune's end it can just be removed then. 

I think there's a stronger argument for having some test that makes sure the scheduler runs, whether it needs to check for reproducibility is less important. For debugging things like `ModelFittingError`it helps to have the code be reproducible, since it only occurs rarely otherwise.

Reply:
Thanks for adapting the test! Could update the name which is now outdated? (for instance to `test_smoke_botorch`)